package.js

```diff
@@ -34,8 +34,10 @@ Package.onTest(function (api) {
   api.use('nimble:restivus');
   api.use('http');
   api.use('coffeescript');
+  api.use('underscore');
   api.use('accounts-base');
   api.use('accounts-password');
+  api.use('mongo', ['client', 'server'])
```

Comment:
The default is for these to be declared on both the client and server, so you can omit the `['client, 'server']` argument.